aws/modules/deploy/helms/gpii-preferences-custom-values.erb

```diff
@@ -0,0 +1,19 @@
+chart-metadata:
+  name: preferences
+  namespace: gpii
+
+replicaCount: <%= ENV["TF_VAR_cluster_name"].start_with?("stg.","prd.") ? '6' : '2' %>
+
+image:
+  repository: <%= @versions["preferences"].split('@')[0] %>
+  checksum: <%= @versions["preferences"].split('@')[1] %>
+
+issuerRef:
+  <% if ENV["TF_VAR_cluster_name"].start_with?("prd.", "stg.") %>
+  name: letsencrypt-production
+  <% else %>
+  name: letsencrypt-staging
+  <% end %>
+
+dnsNames:
+- preferences.<%= ENV["TF_VAR_environment"] %>.gpii.net
```

Comment:
Use TF_VAR_cluster_name instead (it already has the .gpii.net part calculated).

Reply:
I see that TF_VAR_environment is also used in lots of places – do you want to update them as well?

Reply:
TF_VAR_environment and TF_VAR_cluster_name are different, so no you should not update all of them.

If there are cases that do `#{TF_VAR_environment}.gpii.net` like here, it would be great if you would update those to TF_VAR_cluster_name. However I won't ask you to change anything outside of files you're already changing.

Reply:
I will create separate PR for this.

Reply:
Sounds good.